### PR TITLE
Feature/check notarization status

### PR DESCRIPTION
**Pull request by gbonumore**
##### Checklist

- [ ] Linter status: 100% pass
- [ ] Changes don't break existing behavior
- [ ] Test coverage hasn't decreased

### CODE REVIEW

**Comment by AdrianAddesso** [comment]
@gbonumore the text and video look well on desktop and mobile. Could you specify how the IPFS url is expected to be updated?

File: src/components/staking/Summary.svelte [hunk 1]
```diff
@@ -35,6 +38,11 @@
         label: '',
         bn: new BigNumber(0),
     };
+    let nextRate;
+    let nextCompoundWindow;
+    let isNotarizing;
+    let ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
```
Comment (jordaniza) [inline src/components/staking/Summary.svelte]: Minor point but ethers.constants has the zero address

File: src/components/staking/Summary.svelte [hunk 2]
```diff
@@ -202,6 +212,22 @@
...
     }
+
+    const checkNotarization = async () => {
+        const merkleTreeDistributorContract = getMerkleTreeDistributorContract($eth);
+        const nextCreatedIndex = await merkleTreeDistributorContract.nextCreatedIndex();
+        const currentWindowIndex = nextCreatedIndex.sub(BigNumberEthers.from(1))
+        const currentWindow = await merkleTreeDistributorContract.merkleWindows(currentWindowIndex);
+        isNotarizing = currentWindow.merkleRoot !== ZERO_ADDRESS;
+        snapshotAddress = currentWindow.ipfsHash;
+    }
```
Comment (jordaniza) [inline src/components/staking/Summary.svelte:220]: nice

File: src/components/staking/Summary.svelte [hunk 3]
```diff
@@ -202,6 +212,22 @@
...
+
+    onMount(async () => {
+        const res = await fetch("https://raw.githubusercontent.com/pie-dao/shared-metadata/main/slice-rate.json");
+        const data = await res.json();
+        nextRate = data.nextRate;
+        nextCompoundWindow = new Date(data.nextCompoundWindow).toLocaleDateString();
```
Comment (jordaniza) [inline src/components/staking/Summary.svelte]: Might want to error handle as this could blow up - not sure what the fallback here is.

Comment (Alexintosh) [reply src/components/staking/Summary.svelte]: Good spotted

**Review by jordaniza — APPROVED** [review]

**Review by Alexintosh — APPROVED** [review]

**Comment by AdrianAddesso** [comment]
Merging.

**Pull request merged**